File: Python/PRAK205-2210817210013-Muhammad-Shandy-Dailami.py
import math
tinggi = float(input("Masukkan tinggi: "))
miring = float(input("Masukkan miring: "))
alas = math.sqrt(miring * miring - tinggi * tinggi)
keliling = tinggi + miring + alas
luas = 0.5 * alas * tinggi
print("Alas =", round(alas), "cm")
print("Tinggi =", round(tinggi), "cm")
print("Keliling =", round(keliling), "cm")
print("Luas =", round(luas), "cm^2")

File: Bahasa C/PRAK205-2210817210013-Muhammad-Shandy-Dailami.c
#include <stdio.h>
#include <math.h>

int main(){
    float tinggi, miring, alas, keliling, luas;
    printf("Masukkan tinggi: ");
    scanf("%f", &tinggi);
    printf("Masukkan miring: ");
    scanf("%f", &miring);
    alas = sqrt(miring*miring-tinggi*tinggi);
    keliling = tinggi + miring + alas;
    luas = 0.5 * alas * tinggi;
    printf("Alas = %.f cm\n", alas);
    printf("Tinggi = %.f cm\n", tinggi);
    printf("Keliling = %.f cm\n", keliling);
    printf("Luas = %.f cm^2", luas);

    return 0;
}